zeppelin/hdi_add_zeppelin.sh

```diff
@@ -0,0 +1,220 @@
+#! /bin/bash
+
+#----------------------------------------------------------------
+# THIS SCRIPT ADDS INSTALLS AND STARTS ZEPPELIN ON HBASE CLUSTER 
+#----------------------------------------------------------------
+
+
+#----------------------------------------------------------------
+# PRINT USAGE INFORMATION
+#----------------------------------------------------------------
+
+
+print_usage()
+{
+cat << ...
+Usage:
+$0
+
+Mandatory arguments:
+--------------------
+
+-p
+  Admin password for Ambari
+  
+-c
+  DNS name of HBase Cluster
+-h
+  FQDN of head node host
+
+...
+exit 132
+}
+
+#----------------------------------------------------------------
+# INITIALIZE PARAMETERS
+#----------------------------------------------------------------
+
+AMBARI_PASSWORD=
+CLUSTER=
+ZEPPELIN_CONFIG='{ "zeppelin.interpreters": "org.apache.zeppelin.jdbc.JDBCInterpreter,org.apache.zeppelin.markdown.Markdown,org.apache.zeppelin.shell.ShellInterpreter", "zeppelin.config.fs.dir": "file:///etc/zeppelin/conf/","zeppelin.anonymous.allowed": "true","zeppelin.notebook.storage": "org.apache.zeppelin.notebook.repo.VFSNotebookRepo", "zeppelin.server.port": "9995", "zeppelin.interpreter.config.upgrade": "true"}'
```

Comment:
Is the same for different HDI stacks? 2.5, 2.6?

Reply:
we will "support" only 2.6; not tested on 2.5